test/cpp/ext/otel/grpc_trace_bin_text_map_propagator_test.cc

```diff
@@ -87,16 +92,20 @@ TEST(GrpcTraceBinTextMapPropagatorTest, Inject) {
 
 TEST(GrpcTraceBinTextMapPropagatorTest, Extract) {
   TestTextMapCarrier carrier;
-  std::array<char, 29> trace_bin_val;
-  memcpy(trace_bin_val.data(),
-         "\x00\x00"
-         "0123456789ABCDEF"
-         "\x01"
-         "01234567\x02\x01",
-         29);
+  constexpr char kTraceBinValue[] =
+      "\x00"              // version
+      "\x00"              // field 0
+      "0123456789ABCDEF"  // trace
+      "\x01"              // field 1
+      "01234567"          // span
+      "\x02"              // field 2
+      "\x01";             // flag
+  absl::string_view trace_bin_value(kTraceBinValue, sizeof(kTraceBinValue));
```

Comment:
Suggest using `- 1` here and then using `trace_bin_value` below, rather than constructing the `string_view` twice.

Reply:
doesn't look like i'm using this.. surprised that clangtidy did not complain